### PR TITLE
added spring core task - movie theater

### PR DESCRIPTION
**Pull request by tarasvladyka**

### CODE REVIEW

File: movie-theater/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/target/
+
+.classpath
+.project
+.settings/
```
Comment (viktorpr14) [inline movie-theater/.gitignore:5]: Please do not commit local specific files to repository (such as .gitignore etc)

File: movie-theater/src/main/java/com/epam/spring/core/movietheater/service/AuditoriumService.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	public AuditoriumService(AuditoriumDao auditoriumDao) {
+		this.auditoriumDao = auditoriumDao;
+	}
+	public Auditorium getByName(String name) {
+		return auditoriumDao.getByName(name);
+	}
+	public List<Auditorium> getAll(){
+		return auditoriumDao.getAll();
+	}
+}
```
Comment (viktorpr14) [inline movie-theater/src/main/java/com/epam/spring/core/movietheater/service/AuditoriumService.java]: Please use "coding to interface" principal. Best practice is to have interface **AuditoriumService** and than class **AuditoriumServiceImpl implements AuditoriumService**. You should create folder **impl** in folder **service** and all your ...Impl classes should be there

**Review by viktorpr14 — COMMENTED** [review]
Please avoid using "-" in names (such as **movie-theater**, **spring-core**)

File: movie-theater/src/main/java/com/epam/spring/core/movietheater/service/AuditoriumService.java (hunk 2 above)
Comment (tarasvladyka) [reply movie-theater/src/main/java/com/epam/spring/core/movietheater/service/AuditoriumService.java]: project updated according ti this principal

**Pull request merged**